tests/pagination/test_page_position_argument_inspector.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+from mock import Mock
+
+from winter.controller import ControllerMethod
+from winter.controller import ControllerMethodArgument
+from winter.pagination import PagePosition
+from winter.pagination import PagePositionArgumentInspector
+from winter.pagination import PagePositionArgumentResolver
+
+
+
+@pytest.mark.parametrize(('argument_type', 'expected_count_parameters'), (
+        (PagePosition, 2),
+        (object, 0),
+))
+def test_page_position_argument_inspector(argument_type, expected_count_parameters):
+    resolver = PagePositionArgumentResolver()
+    inspector = PagePositionArgumentInspector(resolver)
+    controller_method = Mock(spec=ControllerMethod)
+    argument = Mock(spec=ControllerMethodArgument)
```

Comment:
It's better to use `spec_set` instead of `spec`.
But in this case, it's even better to not use Mock at all. Why not to instantiate `ControllerMethod` and `ControllerMethodArgument` directly?
```
def func(arg1: argument_type):
    pass
controller_method = ControllerMethod(func, '/', 'GET')
argument = controller_method.get_argument('arg1')
```
It's not more verbose and not time-consuming. It will become more sane once routing info will be moved out from ControllerMethod.

If you agree, fix it in all other similar cases.